src/index.ts

```diff
@@ -195,6 +201,11 @@ builder = builder
 void (async () => {
   try {
     const args = await builder.argv;
+    trackEvent('cli_command_success', {
```

Comment:
is it ok that failed commands won't be tracked?

Reply:
I tried, but in the `.fail()` callback I don't see the parameters that I set, they are all `undefined`. I tried a few other ways, nothing worked.

These calls will be tracked in the usual `cli_started` event I guess.

Reply:
Tried that too. The `await builder.argv` line throws an exception which describes the failed request.

Reply:
that is exaclty what we need, no?

Reply:
We need `projectId` and `branchId` and the `AxiosError` doesn't help with that.